lvgl_touch/Kconfig

```
@@ -548,4 +484,31 @@ menu "LVGL Touch controller"
 
     endmenu
 
+    choice
+        prompt "Select an I2C port for the touch panel"
+        default LV_I2C_TOUCH_PORT_0
+        depends on LV_I2C_TOUCH
+
+        config LV_I2C_TOUCH_PORT_0
+            bool
+            prompt "I2C port 0"
+
+        config LV_I2C_TOUCH_PORT_1
+            bool
+            prompt "I2C port 1"
```

Provide help about I2C ports
```suggestion
            prompt "I2C port 1"
            help
                I2C is shared peripheral managed by I2C_Manager. In order to configure I2C_manager (pinout, etc.) see menu
                Component config->LVGL ESP Drivers->I2C Port Settings.
```